src/components/Login/Login.css

```diff
@@ -12,30 +15,72 @@
   height: 100%;
   display: flex;
   justify-content: space-around;
+  align-items: center;
   flex-direction: column;
+  background-image: url("https://images.unsplash.com/photo-1562828536-43a951c08717?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1651&q=80");
+  background-position: center;
+  background-size: cover;
+}
+
+.login-header {
+  margin-top: 0px;
+  margin-bottom: 5%;
+  font-size: 1.75em;
 }
 
 .login-input-field {
   width: 40%;
-  height: 60%;
+  height: 70%;
   display: flex;
   flex-direction: column;
   justify-content: center;
   align-items: center;
-  background-color: grey;
+  margin-top: 5%;
+  border-radius: 5px;
+  border: 2px solid white;
+  opacity: 0.8;
+  background-color: black;
+  
 }
 
 .login-input {
   width: 50%;
   height: 10%;
   font-size: 1.5em;
   margin: 4%;
+  border-radius: 5px;
+  border: 2px solid white;
+}
+
+.btn {
```

Comment:
Forgot you prefer this naming convention for buttons - I will go back in and change mine because I like that better!